### PR TITLE
add bwa to integration tests

### PR DESCRIPTION
**Pull request by nh13**

### CODE REVIEW

**Comment by nh13** [comment]
@jtratner this is ready for review.  Instead of installing conda, and dealing with all that, I just installed BWA manually to get the `bwa` tests to run.

File: .github/workflows/pythonpackage.yml [hunk 1]
```diff
@@ -10,13 +10,21 @@ jobs:
...
       matrix:
-        PYTHON_VERSION: [3.6.10, 3.7.6, 3.8.2]
+        PYTHON_VERSION: [3.6.10, 3.7.7, 3.8.2]
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python ${{matrix.PYTHON_VERSION}}
       uses: actions/setup-python@v1
       with:
         python-version: ${{matrix.PYTHON_VERSION}}
+    - name: Install bwa
```
Comment (pkaleta) [inline .github/workflows/pythonpackage.yml:20]: Isn't the user of this library going to need bwa anyways? Or is the expectation that he should install it by hand?

Comment (nh13) [reply .github/workflows/pythonpackage.yml:20]: The user needs to have bwa to run it.  This is for the integration tests run on github.com

File: .github/workflows/pythonpackage.yml [hunk 2]
```diff
@@ -10,13 +10,21 @@ jobs:
...
+      run: |
+        wget https://github.com/lh3/bwa/releases/download/v0.7.17/bwa-0.7.17.tar.bz2
+        tar -jxvf bwa-0.7.17.tar.bz2
+        cd bwa-0.7.17
+        make -j$(nproc)
+        cd ..
+        echo "::add-path::${GITHUB_WORKSPACE}/bwa-0.7.17/"
```
Comment (jtratner) [inline .github/workflows/pythonpackage.yml:27]: wow, github actions.. totally new language with weird idiosyncracies... https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#adding-a-system-path

I wonder if it detects the `add-path` echo or if random scripts can echo that and it'll run that command?

**Review by jtratner — APPROVED** [review]

**Pull request merged**